programmers/위장.js

```diff
@@ -0,0 +1,15 @@
+function solution(clothes) {
+    const obj = clothes.reduce(function(cloth, kind) {
```

Comment:
```suggestion
    const clothesObj = clothes.reduce(function(cloth, kind) {
```
변수와 함수명 등은 용도를 알기 쉽게 자세히 명명해야 해요.
obj 보다는 clothesObj등이 더 좋을 것 같아요.